doc/classes/EditorExportPlugin.xml

```diff
@@ -158,6 +158,30 @@
 				Return a [PackedStringArray] of additional features this preset, for the given [param platform], should have.
 			</description>
 		</method>
+		<method name="_get_export_option_overrides" qualifiers="virtual const">
```

Comment:
nit: `s` missing at the end of `option`

Reply:
Ah, good catch, thanks! That's the cause of the CI failure about docs. Fixed in my latest push